bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/async/CallController.java

```diff
@@ -84,7 +84,7 @@ public void onNext(RequestT value) {
    *
    * @see com.google.cloud.bigtable.grpc.scanner.RowMerger#onError(Throwable)
    */
-  public void onError(Throwable t) {
+  public synchronized void onError(Throwable t) {
```

Comment:
Please add a comment why this needs to synchronized (that the retry thread calling setCallAndStart is racing with the caller thread invoking request() & cancel())